pandas/_libs/tslibs/strptime.pyx

```diff
@@ -396,7 +401,15 @@ def array_strptime(
 
             result_timezone[i] = tz
 
-        except (ValueError, OutOfBoundsDatetime):
+        except OutOfBoundsDatetime as ex:
```

Comment:
Rather than breaking off this branch can we not keep the existing one and just specialize for the OOB exception within?

Reply:
you're right, that's better - thanks!